kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -0,0 +1,76 @@
+### Domain lifecycle sample scripts
+
+The operator provides sample scripts to start up or shut down a specific Managed Server or cluster in a deployed domain, or the entire deployed domain.
+
+**Note**: Prior to running these scripts, you must have previously created and deployed the domain.
+
+These scripts can be helpful when scripting the life cycle of a WebLogic Server domain. For information on how to start, stop, restart, and scale WebLogic Server instances in your domain, see [Domain Life Cycle](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle).
+
+#### Scripts to start and stop a Managed Server
+The `startServer.sh` script starts a Managed Server either by increasing the `spec.clusters[<cluster-name>].replicas` value for the Managed Server's cluster by `1` or by updating the `spec.managedServers[<server-name>].serverStartPolicy` attribute of the domain resource or both as necessary. The script provides an option to keep the `spec.clusters[<cluster-name>].replicas` value constant for clustered servers. See the script `usage` information by using the `-h` option.
+
+Use the following command to start the server along with increased replica count:
+```
+$ startServer.sh -d domain1 -n weblogic-domain-1 -s managed-server1
+[INFO] Updating replica count for cluster 'cluster-1' to 1.
+domain.weblogic.oracle/domain1 patched
+[INFO] Successfully updated replica count for cluster 'cluster-1' to 1.
+```
+
+Use the following command to start the server without increasing replica count:
```

Comment:
without increasing replica count -> without increasing the replica count

Reply:
Fixed.

Reply:
Changed to say "stop the server either by decreasing the replica count or by updating the server start policy"